packages/ic-response-verification/tests/v2_response_verification_sad_path.rs

```diff
@@ -344,9 +348,13 @@ mod tests {
             MAX_CERT_TIME_OFFSET_NS,
             &root_key,
             MIN_REQUESTED_VERIFICATION_VERSION,
-        );
+        )
+        .unwrap();
 
-        assert!(result.is_err());
+        assert!(matches!(result, VerificationResult::Failed {
+            verification_version,
+            reason: _,
```

Comment:
Same question here regarding adding the reason as a parameter

Reply:
Added 👍  for the `CertificateTimeTooFarInThePast` & `CertificateTimeTooFarInTheFuture` i've ignored the values and just checked if those are the failure reasons. Do you think it would add any value checking the exact times of those errors?

Reply:
That's good! We have unit tests to cover the exact times of those errors so I don't think we need to repeat it here 👍